CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# dbt_ad_reporting v1.4.1
```

Comment:
Since we will be merging this into the next release (which includes breaking changes), I would request we update the version to be inline with the breaking changes within `release/v1.5.0`.
```suggestion
# dbt_ad_reporting v1.5.0
```

Reply:
Would you be able to make the updates to the dbt_project.yml files as well to reflect the new version.

Reply:
Yep, done! 👍